packages/dotnet-sdk/src/TeamsFx/Conversation/NotificationOptions.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace Microsoft.TeamsFx.Conversation
+{
+    /// <summary>
+    /// Options to initialize <see cref="NotificationBot"/>.
+    /// </summary>
+    public class NotificationOptions
+    {
+        /// <summary>
+        /// Whether to enable notification or not. Default true.
+        /// </summary>
+        public bool Enabled { get; set; } = true;
```

Comment:
Why default true? In TS SDK, this is default to false.

Reply:
TS SDK makes `Enabled` out of this `NotificationOptions`, but only in `ConversationOptions`.
Here (.net) has no combined type, so making `Enabled` inside `NotificationOptions`, then I suggest default true since "*developer who creates `NotificationOptions` probably needs notification functionality*".

Reply:
"developer who creates NotificationOptions probably needs notification functionality".
If so, then since we don't need an "Enabled" flag in the `NotificationOptions`, since we are not expect the case that user created a `NotificationOptions` but set the `Enabled` to `false`?

Reply:
Yep... this looks weird, but want to align with `CommandOptions`. Will `CommandOptions` include `Enabled`?
- If yes, `NotificationOptions` should include as well.
- If no, does that mean an empty `CommandOptions` is acceptable?

Reply:
For me, looks like remove `Enabled` would be more straightforward:
- This make the `NotificationBot`/`CommandBot` initialization more clean. (we don't need to check the `Enabled` in the constructor)
- This also simplifies the `ConversationBot` initialization. if user doesn't need to enable notification/command feature, then they don't need to create a `NotificationOptions`/`CommandOptions`, even a empty  `NotificationOptions`, `CommandOptions`.

Reply:
Removed `Enabled`.